### PR TITLE
Error when importing config yml

### PR DESCRIPTION
**Pull request by kelvanb97**

### CODE REVIEW

File: packages/back-end/src/services/organizations.ts [hunk 1]
```diff
@@ -346,6 +346,13 @@ function validateConfig(config: ConfigFile, organizationId: string) {
   const errors: string[] = [];
 
   const datasourceIds: string[] = [];
+  if (!config.datasources)
+    errors.push(
+      `Missing object \`datasources\` check your config.yml validity`
+    );
+  if (!config.metrics)
+    errors.push(`Missing object \`metrics\` check your config.yml validity`);
+
```
Comment (jdorn) [inline packages/back-end/src/services/organizations.ts:355]: Thinking about this more, importing does a merge with your existing configuration, so I can imagine some scenarios where you would only want to import dimensions for example and not datasources or metrics.  So maybe just remove these checks.

File: packages/front-end/components/Settings/RestoreConfigYamlButton.tsx [hunk 2]
```diff
@@ -17,6 +17,8 @@ import cloneDeep from "lodash/cloneDeep";
 import UploadConfigYml from "./UploadConfigYml";
 
 function sanitizeSecrets(d: DataSourceInterfaceWithParams) {
+  if (!d || !d.params) return;
```
Comment (jdorn) [inline packages/front-end/components/Settings/RestoreConfigYamlButton.tsx:20]: This fixes that one specific error, but I'm sure there are countless others that may be caused by different typos.  Can we wrap the entire parsing process in a try/catch so it fails gracefully?  Right now on production, it completely breaks the app and shows a blank white screen.

Comment (kelvanb97) [reply packages/front-end/components/Settings/RestoreConfigYamlButton.tsx:20]: Makes sense to me, I will make that happen.

File: packages/back-end/src/services/organizations.ts (hunk 1 above)
Comment (kelvanb97) [reply packages/back-end/src/services/organizations.ts:355]: I see what you are saying. I am in agreement in that case then. Maybe in the future, this type of check needs to be added after the file is merged if it causes people issues.

**Pull request merged**